src/containers/scenes/mobile/nrc-landing-scene-mobile/nrc-landing-scene-mobile.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+
+import { LANDING } from 'router';
+
+import { aoiAnalyticsActions } from 'actions/google-analytics-actions';
+import * as urlActions from 'actions/url-actions';
+
+import { wrap } from 'popmotion';
+
+import { MARINE_SPI_FEATURE_LAYER } from 'constants/layers-slugs';
+
+import Component from './nrc-landing-scene-mobile-component';
+import { NRC_LANDING_CARDS } from './nrc-landing-scene-mobile-constants';
+
+const actions = { ...urlActions, ...aoiAnalyticsActions };
+
+function NrcLandingSceneMobileContainer(props) {
+  const { activeLayers, browsePage } = props;
+
+  const [[page, direction], setPage] = useState([0, 0]);
+
+  const [selectedLayers, setSelectedLayers] = useState(activeLayers);
+
+  const cardIndex = wrap(0, NRC_LANDING_CARDS.length, page);
```

Comment:
🤔 I think we can use the % [remainder](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder) instead of importing a new module